### DISCUSSION TITLE
Vue compatibility

### DISCUSSION THREAD

**Discussion by realsamanrad** [emdgroup-liquid/liquid#456] [Q&A] [answered]
Hi. Is there any procedure for using with Vue/Nuxt ?
Thanks

**Comment by borisdiakur** [comment]
Hi @samanrd 
I'm happy you asked this question, because I like working with Vue myself and would like to make Liquid as friendly as possible for Vue developers. That said, we've been concentrating with framework specific bindings on React so far. I would like to pick up this topic and see if it makes sense to also integrate [Vue bindings](https://stenciljs.com/docs/vue) in the project. That should improve the DX for Vue developers. For now, you can still use Liquid without any framework specific bindings in a Vue project. We have a couple of [sandbox applications](https://liquid.merck.design/liquid/guides/sandbox-applications/) in our docs which demonstrate how you can use Liquid in different tech stacks. You'll find a Vue app at the top of the list. The project is using Vue's `defineComponent` with JSX in order to have type checking and intellisense. My hope is that Vue specific bindings would help achieving the same DX with Vue template syntax.

I changed your issue to a discussion for now. Let me know if you think Vue bindings are a must for using Liquid in Vue. Then we'll open up a new issue to track progress on that task. Any thoughts, ideas are welcome.

**Comment by borisdiakur** [comment, marked as answer]
__UPDATE:__ I found some time to work on this topic. The latest Liquid version ([v4.2.0](https://github.com/emdgroup-liquid/liquid/releases/tag/v4.2.0)) now includes [Vue bindings](https://liquid.merck.design/liquid/guides/framework-integrations/vue-bindings/). The [Vue sandbox app](https://stackblitz.com/github/emdgroup-liquid/liquid-sandbox-vue-tailwind) demonstrates how to integrate Liquid in a Vue app using these bindings. Please have a look 👀
I'll try to find some more time to also create another sandbox app using Nuxt. It should be pretty similar to the existing sandbox app.

**Comment by borisdiakur** [comment]
We now have also a [Nuxt based sandbox app](https://stackblitz.com/github/emdgroup-liquid/liquid-sandbox-nuxt-tailwind).

You may notice that it includes a [patch](https://stackblitz.com/github/emdgroup-liquid/liquid-sandbox-nuxt-tailwind?file=patches%2F%40emdgroup-liquid%2Bliquid%2B4.2.0.patch%3AL10): There is an issue with Nuxt and the vue output target. It's not a deal breaker, but I have [already reported](https://github.com/ionic-team/stencil-ds-output-targets/issues/319) it to the stencil team anyway.
